scripts/npm-stats.ts

```diff
@@ -39,21 +45,28 @@ async function getAllStats(page) {
     '@sap-cloud-sdk/temporal-de-serializers',
     '@sap-cloud-sdk/odata-common',
     '@sap/cds'
-  ].reduce(
-    async (stats, packageName) => ({
-      ...(await stats),
-      [packageName]: await getStatsForPackage(page, packageName)
-    }),
-    {}
-  );
+  ].map(packageName => {
+    return getStatsForPackage(browser, packageName);
```

Comment:
Why did you remove the `await` here? Other than that LGTM

Reply:
The function returns an array of promises so that they can run in parallel. Reduce would run in sequence and await each response before continuing to the next element.